### DISCUSSION TITLE
Can I set "copy_df" value in ipsec.conf

### DISCUSSION THREAD

**Discussion by Airren** [strongswan/strongswan#773] [Q&A] [answered]
I know since 5.7.0，"New options in swanctl.conf allow configuring how/whether DF, ECN and DS fields in the IP headers are
copied during IPsec processing."  
Since this version, in swanct.conf we allowed to customize` connections.<conn>.children.<child>.copy_df.`
But now our project still uses` ipsec.conf` to set the tunnel configuration, I want to know can we customize  `copy_df` value in` ipsec.conf`.

**Comment by tobiasbrunner** [comment, marked as answer]
No, you can't. The `ipsec.conf`-based configuration backend is deprecated and has not received any new features for several years.
